os/include/tinyara/sensors/sensor.h

```diff
@@ -0,0 +1,431 @@
+/****************************************************************************
+ *
+ * Copyright 2017 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+/**
+ * @file sensor.h
+ * @brief Sensor API definition
+ * @version 0.3
+ */
+
+#ifndef __SENSOR_H__
```

Comment:
Please follow our style. __PATH_NAME_H
e.g. for os/include/tinyara/sched.h => __INCLUDE_TINYARA_SCHED_H